daxa/process/erosita/assemble.py

```diff
@@ -0,0 +1,202 @@
+# This code is a part of the Democratising Archival X-ray Astronomy (DAXA) module.
+# Last modified by Jessica Pilling (jp735@sussex.ac.uk) Wed Jul 19 2023, 13:52. Copyright (c) The Contributors
+from random import randint
+import os.path
+
+from astropy.units import Quantity
+
+from daxa import NUM_CORES
+from daxa.archive.base import Archive
+from daxa.process._cleanup import _last_process
+from daxa.process.erosita._common import _esass_process_setup, ALLOWED_EROSITA_MISSIONS, esass_call, _is_valid_flag
+
+@_last_process(ALLOWED_EROSITA_MISSIONS, 1)
+@esass_call
+def cleaned_evt_lists(obs_archive: Archive, lo_en: Quantity = Quantity(0.2, 'keV'), hi_en: Quantity = Quantity(10, 'keV'),
+                      flag: int = 0xc0000000, flag_invert: bool = False, pattern: int = 15, num_cores: int = NUM_CORES,
+                      disable_progress: bool = False, timeout: Quantity = None):
+
+    """
+    The function wraps the eROSITA eSASS task evtool, which is used for selecting events.
+    This has been tested up to evtool v2.10.1
+
+    This function is used to apply the soft-proton filtering (along with any other filtering you may desire, including
+    the setting of energy limits) to eROSITA event lists, resulting in the creation of sets of cleaned event lists
+    which are ready to be analysed (or merged together, if there are multiple exposures for a particular
```

Comment:
Removed this now since the function can't do this!

I think I had left it in because evtool can merge event lists into one big one, in the case of people wanting to merge neighbouring sky tiles from eRASS into one file (was my impression from the EDR website). So I thought that it would be good to have that functionality in DAXA. 

Should I maybe add an Issue to implement this later? I wasn't sure if it would be handled more on the DAXA or XGA side of things?